index.js

```diff
@@ -3,8 +3,12 @@ const openNavButton = document.getElementById("open-nav-button");
 const closeNavButton = document.getElementById("close-nav-button");
 const addTaskButton = document.getElementById("add-task-button");
 const taskTitleInput = document.getElementById("task-title-input");
+const taskTable = document.getElementById("task-table");
+const TaskItems = document.createElement("tr");
```

Comment:
JavaScript variables by convention should be camel case.

Reply:
Thanks for your comment.
I'll try not to repeat these mistakes.
But I changed the whole algorithm.
Hope it will be correct.